scripts/pethtool

```diff
@@ -56,7 +56,7 @@ def usage():
         [ tso on|off ]''')
 
 
-tab = ""
+tab = ''
```

Comment:
Can you do quotation marks unification with some bogus commit author as I did with PEP7/8? So that you don't end up all over git blame?